### PR TITLE
Add MYSQL Feature

### PR DESCRIPTION
**Pull request by michaels1999**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>io.angelwing</groupId>
-    <artifactId>car-rental-service</artifactId>
+    <artifactId>wallet-app</artifactId>
```
Comment (dcadea) [inline pom.xml:8]: Revert this

File: pom.xml [hunk 2]
```diff
@@ -18,16 +18,37 @@
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
     </properties>
+    <!-- https://mvnrepository.com/artifact/org.hibernate/hibernate-core -->
```
Comment (dcadea) [inline pom.xml:21]: this comment is redundant, please remove

File: pom.xml [hunk 3]
```diff
@@ -18,16 +18,37 @@
...
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>runtime</scope>
```
Comment (dcadea) [inline pom.xml:40]: use h2 for tests only, the scope should be `test`

File: src/main/java/io/angelwing/car/rental/service/model/Car.java [hunk 4]
```diff
@@ -1,21 +1,39 @@
...
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 import java.util.UUID;
 
+@Entity
```
Comment (dcadea) [inline src/main/java/io/angelwing/car/rental/service/model/Car.java:13]: Hibernate requires all POJOs to delare a default constructor.

**Review by dcadea — CHANGES_REQUESTED** [review]

File: src/main/java/io/angelwing/car/rental/service/CarRentalApplication.java [hunk 5]
```diff
@@ -11,4 +14,11 @@ public static void main(String[] args) {
         SpringApplication.run(CarRentalApplication.class, args);
     }
 
+    @Bean
+    public CommandLineRunner run() {
```
Comment (dcadea) [inline src/main/java/io/angelwing/car/rental/service/CarRentalApplication.java:18]: not critical, but this code is not used

Comment (michaels1999) [reply src/main/java/io/angelwing/car/rental/service/CarRentalApplication.java:18]: I already deleted this paragraph

**Pull request merged**